src/Microsoft.ML.FastTree/RandomForestClassification.cs

```diff
@@ -10,7 +10,6 @@
 using Microsoft.ML.CommandLine;
 using Microsoft.ML.Data;
 using Microsoft.ML.EntryPoints;
-using Microsoft.ML.Internal.Calibration;
 using Microsoft.ML.Internal.Internallearn;
```

Comment:
I assume `Internal.Internallearn` will be taken care of in another change.

Reply:
Yep, I’ll be taking care of that today.